daemon/src/devices/pinetimejfdevice.cpp

```diff
@@ -10,9 +10,39 @@
 #include "hrmservice.h"
 #include "infinitimemotionservice.h"
 #include "infinitimeweatherservice.h"
-
+#include "adafruitblefsservice.h"
 #include <QtXml/QtXml>
 
+namespace {
+    size_t GetMtuForCharacteristic(QString path, const char* characteristicUUID)
```

Comment:
I think this should be moved to qble/qbledevice?

Reply:
I put this method in PinetimeJFDevice because it already contains some code that queries DBus to do the service discovery. Maybe we should move all this code to qble?

Reply:
So, yes, you are right, this should be moved to qble/qbledevice, but I think this change will have impact in other device implementations that also do the service discovery. Is that OK if we do that in another PR?

Reply:
ok